src/content/developers/docs/nodes-and-clients/index.md

```diff
@@ -19,18 +19,50 @@ If you're new to the topic of nodes, we recommend first checking out our user-fr
 
 ## What are nodes and clients? {#what-are-nodes-and-clients}
 
-A "node" is a computer running Ethereum client software. A client is an implementation of Ethereum that verifies all transactions in each block, keeping the network secure and the data accurate. Up until The Merge, a single piece of software is required to run a full node (or two to run a mining node). After The Merge, two pieces of client software are required to run a full node (three to run a validator node), one to handle and gossip transactions (execution client), one to handle block gossip and fork choice (consensus client) and an optional validator client that handles block production and "voting" on blocks received from its peers.
+"Node" refers to a running piece of client software. A client is an implementation of Ethereum that verifies all transactions in each block, keeping the network secure and the data accurate.
```

Comment:
> "Node" refers to a running piece of client software.

This sentence doesn't make sense. IMO here we should describe **what a node is** and **not what running a node is** (as per the heading)

Reply:
Makes sense. Changing to `"Node" refers to a running piece of client software.`

Reply:
Fair enough, I am wondering how to define it properly. What would you suggest?  
"Running" here is meant as client software which is running/executed somewhere, connected to the network, etc, not the act of somebody running it. Maybe something like `A "node" is an instance of Ethereum client participating in the network.`

Reply:
Yeah, fundamentally a node is Ethereum software being run on a computer, and that software communicates with others doing the same. 

Perhaps this?
`A "node" is any instance of Ethereum client software that is connected to other computers also running Ethereum software, forming a network.`

> A client is an implementation of Ethereum that verifies all transactions in each block, keeping the network secure and the data accurate.

Note here not all clients deal with verifying transactions, ie consensus client, as you get into below... Maybe we make this a little more agnostic?
`A client is an implementation of Ethereum that verifies data against the protocol rules and keeps the network secure.`

Reply:
@wackerow Sounds good to me!